src/Extensions/Shopware/PluginCreator/template/current/Subscriber/ControllerPath.tpl

```diff
@@ -64,6 +64,7 @@ class ControllerPath implements SubscriberInterface
      */
     public function onGetControllerPathFrontend(\Enlight_Event_EventArgs $args)
     {
+        $this->container->get('template')->addTemplateDir(__DIR__ . '/..' . '/Resources/views/');
```

Comment:
why not `__DIR__ . '/../Resources/views/'` ?

Reply:
I just made it the same like some lines above
https://github.com/Gugiman/sw-cli-tools/blob/8b409dc9c1721a74e0f1698b5fdedb6857b06385/src/Extensions/Shopware/PluginCreator/template/current/Subscriber/ControllerPath.tpl#L52